init.sql

```diff
@@ -25,6 +25,38 @@ CREATE INDEX cl_items_node_idx on cl_items (node_idx);
 CREATE INDEX cl_items_leaf_idx on cl_items (leaf_idx);
 CREATE UNIQUE INDEX cl_items__tree_node on cl_items (tree, node_idx);
 
+CREATE TABLE backfill_items
+(
+    id          bigserial PRIMARY KEY,
+    tree        BYTEA  NOT NULL,
+    seq         BIGINT NOT NULL,
+    slot        BIGINT NOT NULL,
+    force_chk   bool
+);
+
+CREATE INDEX backfill_items_tree_idx on backfill_items (tree);
+CREATE INDEX backfill_items_seq_idx on backfill_items (seq);
+
+CREATE or REPLACE FUNCTION notify_new_backfill_item()
+    RETURNS trigger
+     LANGUAGE 'plpgsql'
+as $BODY$
+declare
+begin
+    if (tg_op = 'INSERT') then
```

Comment:
So far from my basic research it seems like triggering procedures cause issues if you are incorrectly doing complex relational integrity things and modifying your data.  Triggers are global and they always run regardless of circumstances.  Also nested procedures, recursive ones, and similar, get complicated and hard to debug.  However, it still seems like some high performance database management systems use them for things like auto-generated columns.

But my use case is exceedingly simple.  I'm not modifying any data with this, or even counting on this event to refer to a specific INSERT event.  There's no data integrity implications as I'm using the trigger as a semaphore to wake up a sleeping task.

It seems like it would just be a question of overall load performance issue for the database, and since this procedure is so minimal (it just calls a single `pg_notify()` method), I'm doubting it will bog the system down in any way.

Plus, even if we miss an event, we will just be triggered next time as well.  There's no coupling between the trigger and the trees to backfill.  I will keep looking into this topic more.